rules/etcd-encryption-native/raw.rego

```diff
@@ -1,23 +1,21 @@
 package armo_builtins
+
 import data.cautils as cautils
 
 # Check if encryption in etcd is enabled for native k8s
 deny[msga] {
 	apiserverpod := input[_]
```

Comment:
Where is the api server pod validation?

Reply:
that rule was not a part of my changes and it is not part of the CIS

Reply:
So why is it in the changes?